src/protagonist/DLCS.AWS/SNS/ITopicPublisher.cs

```diff
@@ -1,4 +1,4 @@
-﻿using DLCS.Model.Messaging;
+﻿using DLCS.Model.Customers;
```

Comment:
what was the reason for moving this to `Customer`? Mainly asking as there's methods in here for publishing to the `AssetModified` topic

Reply:
It hasn't moved - this is just updating a reference. The new `CustomerCreatedNotification` takes a `Customer` object. `using DLCS.Model.Messaging` was redundant.